dashboard/yarn.lock

```diff
@@ -0,0 +1,17632 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
```

Comment:
Can we remove this file?

Reply:
ok！I will delete yarn.lock , and add to the ignore file